src/resolvers/hostTags/index.ts

```diff
@@ -80,9 +82,18 @@ export default async function hostTags(parent: any, args: QueryHostTagsArgs, con
             return value;
         }
 
+        let cased_tag: string;
+
+        // We need to prune of the case insensitive part if we used a Regex query
+        if (bucket.key.includes('|')) {
+            cased_tag = split(bucket.key, '|')[0];
```

Comment:
I know we never allow `\` in our namespaces, but what about `|`? If the key already contains `|`, this would break. Not that we should encourage the use of `|` in our tags...

Reply:
Good point! Given how split works improperly partitioning this could cause the service to throw, and that would be quite bad. I'm thinking of using something astronomically unlikely like `{C|L}`, but I may need to alter the split function to get it working. I figure several characters in a specific order like that is basically never going to happen so I think it will be fine.

Reply:
Pushed the fix, all the test are passing so I think we're good :+1: